src/main.ts

```diff
@@ -217,8 +209,8 @@ const createWindow = async () => {
   configureWatcher(mainWindow)
   wasAppClosedByClient = false
 
-  proxyEmitter.on('status:change', (status: ProxyStatus) => {
-    proxyStatus = status
+  k6StudioState.proxyEmitter.on('status:change', (status: ProxyStatus) => {
```

Comment:
Variables that were once global within the `main.ts` file can now be accessed via `k6StudioState` in the entire main process.